lib/config.ts

```diff
@@ -71,16 +75,31 @@ export class Config {
   /**
    * @todo Make private?
    */
-  _init() {
+  _init(): void {
     if (process.env.MSB_CONFIG_PATH) {
       const configPath = resolve(process.env.MSB_CONFIG_PATH);
       const jsonObj = require(configPath);
       delete(require.cache[configPath]);
       this.configure(jsonObj);
     }
   }
+
+  private sslValue(): boolean {
```

Comment:
I'm not sure that we need such a comprehensive method for getting ssl value. 
At least, we can take out it to utils (e.g. getBoolean()) and use this method for all boolean variables in future.